dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/action/EditContentletAction.java

```diff
@@ -1294,16 +1333,27 @@ else if(perAPI.doesUserHavePermission(contentlet, PermissionAPI.PERMISSION_WRITE
 			Matcher m = p.matcher(interval);
 			boolean b = m.matches();

```

Comment:
rename "m" and "b" to matcher and matches

Reply:
done